pkg/storage/gc/gc.go

```diff
@@ -338,9 +372,9 @@ func (gc GarbageCollect) removeManifest(repo string, index *ispec.Index,
 				return false, err
 			}
 		} else {
-			err := gc.metaDB.RemoveRepoReference(repo, desc.Digest.String(), desc.Digest)
+			err := gc.metaDB.RemoveRepoReference(repo, reference, desc.Digest)
 			if err != nil {
-				gc.log.Error().Err(err).Msg("gc, metadb: unable to remove repo reference in metaDB")
+				gc.log.Error().Err(err).Msg("gc,metadb: unable to remove repo reference in metaDB")
```

Comment:
Maybe update all the logs in package `gc` to add the `module` value:
```suggestion
				gc.log.Error().Err(err).Str("module","gc").Msg("unable to remove repo reference in metaDB")
```